src/types.ts

```diff
@@ -29,4 +29,79 @@ export type SafeTransactionData = {
   gasPrice?: bigint
   gasToken?: Address
   refundReceiver?: Address
+}
+
+export type SafeInfoResponse = {
+  readonly address: string
+  readonly nonce: number
+  readonly threshold: number
+  readonly owners: string[]
+  readonly masterCopy: string
+  readonly modules: string[]
+  readonly fallbackHandler: string
+  readonly guard: string
+  readonly version: string
+}
+
+export type SafeMultisigConfirmationResponse = {
+  readonly owner: string
+  readonly submissionDate: string
+  readonly transactionHash?: string
+  readonly confirmationType?: string
+  readonly signature: string
+  readonly signatureType?: string
+}
+
+export type SafeMultisigConfirmationListResponse = {
+  readonly count: number
+  readonly next?: string
+  readonly previous?: string
+  readonly results: SafeMultisigConfirmationResponse[]
+}
+
+export type SafeMultisigTransactionResponse = {
+  readonly safe: string
+  readonly to: string
```

Comment:
Address/Hex properties should be properly typed (as `Address` for instance)

Reply:
Updated to cover Address / Hex where possible. I left string in places where Safe had it for things like dates, threshold etc